vpr/src/device/rr_graph_builder.h

```diff
@@ -59,6 +59,20 @@ class RRGraphBuilder {
     /** @brief Clear all the underlying data storage */
     void clear();
 
+    // Reorder RRNodeId's using one of these algorithms:
```

Comment:
Please follow the comment style of other APIs, which are compatible with Doxygen.
As such, the API comments will be shown in online documentation.